walletunlocker/service.go

```diff
@@ -294,9 +323,11 @@ func (u *UnlockerService) InitWallet(ctx context.Context,
 	// now send over the wallet password and the seed. This will allow the
 	// daemon to initialize itself and startup.
 	initMsg := &WalletInitMsg{
-		Passphrase:     password,
-		WalletSeed:     cipherSeed,
-		RecoveryWindow: uint32(recoveryWindow),
+		Passphrase:      password,
+		WalletSeed:      cipherSeed,
+		RecoveryWindow:  uint32(recoveryWindow),
+		StatelessInit:   in.StatelessInit,
+		MacResponseChan: make(chan []byte),
```

Comment:
should be buffered?

Reply:
Good point! Fixed it and added a comment in both places.